Rust/tokio-echo/README.md

```diff
@@ -0,0 +1,91 @@
+# tokio_echo
+
+This demo is a TCP Echo server for tokio
+
+## Prerequisites
+
+Be sure to update your rust stable toolchain to at least 1.60.0.
+
+Please ensure that you're using the standard toolchain (rustup default).
+
+## Running from Try Enarx
+
+First generate the Wasm file on localhost:
+
+```console
+cargo build --target wasm32-wasi 
+```
+
+Access the __Try Enarx__ website:
+
+https://try.enarx.dev
+
+Select a platform. Let's try Intel SGX on Equinix.
+
+Upload the generated Wasm file from:
+
+```console
+/target/wasm32-wasi/debug/tokio-echo.wasm 
+```
+
+Paste the following to the Enarx.toml text area:
+
+```
+[[files]]
+kind = "stdin"
+
+[[files]]
+kind = "stdout"
+
+[[files]]
+kind = "stderr"
+
+[[files]]
+name = "LISTEN"
+kind = "listen"
+port = 12345
+prot = "tcp"
+```
+
+Click on the `Deploy` button. 
+
+Now open a terminal on your machine and enter:
+
+```console
+echo hello world | ncat sgx.equinix.try.enarx.dev 12345 
```

Comment:
```suggestion
echo hello | ncat sgx.equinix.try.enarx.dev 12345 
```

to avoid possible space handling oddities in different shells (otherwise, quote the string)

Additiionally, there should be documentation here about what `ncat` is and how to install it on different systems. Btw, I think `nc` may be more prevalent, but I'm not sure what's the situation today. @haraldh @platten any ideas what's available on Mac, WSL out-of-the-box?

Ubuntu, Debian, Fedora have neither available out-of-the box:
```
$ docker run --rm -it ubuntu 
root@7032a4d2610f:/# nc
bash: nc: command not found
root@7032a4d2610f:/# ncat
bash: ncat: command not found
exit
$ docker run --rm -it debian
root@5d834cf6f28a:/# nc
bash: nc: command not found
root@5d834cf6f28a:/# ncat
bash: ncat: command not found
exit
$ docker run --rm -it fedora
[root@00a0eb61be77 /]# nc
bash: nc: command not found
[root@00a0eb61be77 /]# ncat
bash: ncat: command not found
```

Reply:
Thanks. I've added a reference to nc as well. Maybe we keep the instructions short on each individual demo, and add more details for all.